packages/buffetjs-styles/src/components/Checkbox/index.js

```diff
@@ -49,6 +54,27 @@ const Checkbox = styled.input`
       display: block;
     }
   }
+
+  ${({ someChecked }) => {
+    if (someChecked) {
+      return `
+        &:after {
+          content: '\f068';
+          position: absolute;
+          display: block;
+          top: 50%;
+          left: 0px;
+          font-size: 10px;
+          font-family: 'FontAwesome';
+          font-weight: 100;
+          color: #1c5de7;
```

Comment:
You can remove all the attributes already declared below like `position`, `left`, `font-family`, `font-weight` and `color`.